src/tools/NeighborList.h

```diff
@@ -78,7 +78,9 @@ class NeighborList
   std::pair<unsigned,unsigned> getClosePair(unsigned i) const;
 /// Get the list of neighbors of the i-th atom
   std::vector<unsigned> getNeighbors(unsigned i);
-  ~NeighborList() {}
+  ~NeighborList() {};
```

Comment:
@SilvioPipolo the `;` at the end of this line is redundant and should be removed I think (compilers give warnings from time to time on this stuff).